### DISCUSSION TITLE
Playlist with RTMP

### DISCUSSION THREAD

**Discussion by ngivoni** [savonet/liquidsoap#2510] [User Support]
Is it possible to create a playlist (or something similar using another process) with files and RTMP playing in a sequential order? Something like the following:

first_file.mp4
second_file.mp4
RTMP stream
third_file.mp4

**Comment by toots** [comment]
Aren't `rtmp` streams potentially infinite? How do you know when to switch from RTMP to `third_file.mp4`? When it disconnects?

**Reply by ngivoni** [reply]
I would provide a duration in which it would stream for or the stream would end/disconnect.

**Comment by toots** [comment]
I think it's important to think about this in terms of automation: you want to direct the script to be able to pull data at any time. Here, for instance, what happens after the third file? And after that? etc.
